src/core/Message.js

```diff
@@ -3,16 +3,14 @@
  * @author Brandon Alexander - baalexander@gmail.com
  */
 
-var assign = require('object-assign');
+import assign from 'object-assign';
 
 /**
  * Message objects are used for publishing and subscribing to and from topics.
  *
  * @constructor
  * @param values - object matching the fields defined in the .msg definition file
  */
-function Message(values) {
+export function Message(values) {
```

Comment:
Even with the ESLint update, the main source code changes are the import statements, not much else changed.